lib/internal/event_target.js

```diff
@@ -229,17 +227,15 @@ class EventTarget {
       throw new ERR_INVALID_ARG_TYPE('event', 'Event', event);
     }
 
-    if (this.#emitting.has(event.type) ||
-        event[kTarget] !== undefined) {
-      throw new ERR_EVENT_RECURSION(event.type);
+    if (event[kTarget] !== undefined) {
```

Comment:
While being on it, should we not also change `kTarget` to `#target`?

Reply:
@BridgeAR kTarget is set from the outside (from the EventEmitter onto the event) so it can't be made a private property.